macos/Flutter/Flutter-Debug.xcconfig

```diff
@@ -0,0 +1,2 @@
+#include? "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
```

Comment:
we don't need `macos` folder structure, at least for now, lets remove it

Reply:
✅